alicloud/table_alicloud_rds_instance.go

```diff
@@ -94,6 +101,20 @@ func tableAlicloudRdsInstance(ctx context.Context) *plugin.Table {
 				Transform:   transform.FromField("DBInstanceClass"),
 				Description: "The instance type of the instances.",
 			},
+			{
+				Name:        "sql_collector_status",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getSqlCollectorPolicy,
+				Transform:   transform.FromField("SQLCollectorStatus"),
+				Description: "The status of the SQL Explorer (SQL Audit) feature.",
+			},
+			{
+				Name:        "audit_retention_duration",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getSqlCollectorRetention,
+				Transform:   transform.FromField("ConfigValue"),
+				Description: "The log backup retention duration that is allowed by the SQL explorer feature on the instance.",
```

Comment:
The audit retention duration that is allowed by the SQL explorer feature on the instance.